.github/workflows/check-pr-title.yaml

```
@@ -0,0 +1,39 @@
+name: Check PR Title
+
+on:
+  pull_request_target:
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  check:
+    name: Check PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5.5.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            chore
+            fix
+            feat
+            revert
+
+          scopes: |
+            api
+            ci
+            deps
+            docs
+            examples
+            exporter
+            operator
+            initializer
+            manifests
+            runtimes
+            scripts
+            test
+          requireScope: false
```

We should skip PRs created by dependabot https://github.com/kubeflow/trainer/pulls?q=is%3Apr+is%3Aclosed+author%3Aapp%2Fdependabot
```suggestion
          requireScope: false
          ignoreLabels: |
            dependencies
```

---

Same with `wip: true`, wdyt?

---

Interesting, let me exclude this label as well.